packages/js/product-editor/src/components/details-categories-field/create-category-modal.scss

```diff
@@ -1,5 +1,6 @@
 .woocommerce-create-new-category-modal {
 	min-width: 650px;
+	overflow: visible;
```

Comment:
Seems odd for a modal to have `overflow: visible` set. What scenario is this needed for?

Reply:
This is needed because we want the values of SelectTreeMenu to appear even outside of the modal's boundaries. It was the best way I found, since using portals interfered with navigation using tab (after trying this I don't think it's a good idea to use portals like how it was done in the storybook example in https://github.com/woocommerce/woocommerce/pull/37574)

Reply:
> This is needed because we want the values of SelectTreeMenu to appear even outside of the modal's boundaries

Interesting, because I am pretty sure I tested removing the `overflow: visible` and things still worked? I must have been confused!